<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_movie</class>
 <widget class="QWidget" name="add_movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: #EEE;
	background-color: #111;
}

#add_movie {
	background-color: qlineargradient(spread:pad, x1:0.637, y1:0.142364, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 86, 86, 207));
}

#title {
	background-color: transparent;
	font-size: 26px;
	color: rgba(13, 83, 170, 0.9);
}

#error {
	background-color: transparent;
	color: rgba(255, 0, 0, 0.8);
}

QLabel {
	font-size: 20px;
}

QLineEdit, QDateTimeEdit {
	border-radius: 10px;
	background-color: rgba(20, 20, 20, 0.75);
	padding: 5px 10px;
	border:  1px solid rgba(255, 255, 255, 0.25);
	border-bottom-color: rgba(110, 110, 255, 0.25);
	font-size: 16px;
}
QLineEdit:focus {
	border-bottom-color: #0d53aa;
}

QPushButton {
	border-radius: 10px;
	padding: 5px 10px;
	border: 1px solid #0d53aa ;
	background-color: #0d53aa; 
	font-weight: 900;
	font-size: 16px;
}
QPushButton:hover {
	background-color: red; 
	border-color: red;
}

#cancel_btn {
	border-color: rgba(84, 84, 84, 0.75);
	background-color: rgba(84, 84, 84, 0.75);
}</string>
  </property>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Movie Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cinemabooking Dashboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="description">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Movie Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posterURL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Poster URL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="seats">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Seats</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genres">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Genres</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="showDate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2100</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUser_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>362</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="editUser_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>434</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Edit User</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="addAdmin_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>506</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Add Admin</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="deleteAdmin_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>578</y>
     <width>133</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Delete Admin</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="addMovie_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>146</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Add movie</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="deleteMovie_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>218</y>
     <width>130</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Delete movie</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="editMovie_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Edit Movie</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>description</tabstop>
  <tabstop>posterURL</tabstop>
  <tabstop>seats</tabstop>
  <tabstop>showDate</tabstop>
  <tabstop>genres</tabstop>
  <tabstop>add_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>logout_btn</tabstop>
  <tabstop>addMovie_page</tabstop>
  <tabstop>deleteMovie_btn</tabstop>
  <tabstop>editMovie_page</tabstop>
  <tabstop>deleteUser_page</tabstop>
  <tabstop>editUser_page</tabstop>
  <tabstop>addAdmin_page</tabstop>
  <tabstop>deleteAdmin_page</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
